public/javascripts/jquery.tokeninput.js

```diff
@@ -824,11 +809,7 @@ $.TokenList = function (input, url_or_data, settings) {
                 .hide();
 
             $.each(results, function(index, value) {
-                var this_li = $("<li>" + highlight_term(escapeHTML(value.name), query) + "</li>")
-                                  .attr({
-                                    "id": hidden_input_id + "_autocomplete_suggestion_" + index,
-                                    "role": "option" // was "menuitem"
-                                  })
+                var this_li = $("<li role=\"option\">" + highlight_term(escapeHTML(value.name), query) + "</li>") // was role=\"menuitem\"
```

Comment:
I'm not 100% convinced adding the `role` attribute with `.attr()` worked right, so I'm just going to leave this with the old ugly version, even though I maintained the switches to `.attr()` elsewhere.